.github/workflows/wheels.yml

```diff
@@ -16,27 +16,19 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.2.2
+        uses: pypa/cibuildwheel@v2.3.1
```

Comment:
The latest release seems to be 2.9.0.
But I looking at the releases [notes](https://github.com/pypa/cibuildwheel/releases), it doesn't seem to me that a big bug was fixed.
I would prefer to update it only when we will also add support to Python 3.11, but not right yet.

What do you think @ulupo ?

Reply:
We only have a simple CI for now, and I'd prefer not to have surprisesn right now. So I agree with you!